src/styles/shared/_buttons.scss

```diff
@@ -201,6 +201,7 @@
   border-color: rgba($outline-color, 0.4);
   box-shadow: none;
   color: darken($outline-color, 20%);
+  @include recolor-icon(darken($outline-color, 20%));
```

Comment:
Be careful of relying on functions like `darken` (see previous comment re: rgba/darken). This is also a smell that a color is not accounted for in our palette/color system, since we have to transform existing colors to create it.